### ISSUE TITLE
Set language

### ISSUE THREAD

**Issue by ran-j** [NaturalNode/natural#463]
hello.

Instead of doing:
```
Portugese | X |   |   | PorterStemmerPt
Russian | X |   |   | PorterStemmerRu
Swedish | X |   |   | PorterStemmerSv
```

Can you create a config ? , like:

`natural.Language('PT-BR');`

and the just use just 

`'strings'.stem()`

**Comment by Hugo-ter-Doest** [comment]
I'm not sure if I understand what you mean. At the moment stemmers for different languages are separate functions. Is it that you can set the language of natural and after that use stemmers like that? So if I say `natural.setLanguage('Pt') that all modules can be used in Portugese? 

Interesting idea. We need to improve the language system of natural, so maybe this is a way to go.

**Comment by ran-j** [comment]
I'm saying to use one function to all languages and set the language that you wanna use example:

```
//set the PT language
natural.Language('PT-BR');

//and the use the function:
'strings'.stem()
```

To set the language that I want to use dynamic.

```
//set the PT language
natural.Language('PT-BR');

//and the use the function:
'strings'.stem()

//set the EN language
natural.Language('EN-ES');

//and the use the function:
'strings'.stem()
```

**Comment by Hugo-ter-Doest** [comment]
It seems that you are using ISO language codes. PT-BR is Brazilian Portuguese. But what is EN-ES, or do you mean EN-US?

**Comment by ran-j** [comment]
Yes kkk I mean EN-US was just a example. 

Did you get the idea ?

**Comment by Hugo-ter-Doest** [comment]
Yeah I get the idea. It will take quite some refactoring of modules that support multiple languages. Plus I'm afraid we have to introduce a global setting that can be seen by all modules. Something like:
```
function setLanguage(l) {
  global.language = l;
}
```

**Comment by ran-j** [comment]
yes like that, The code will get more organized and easy to change the language. Im sking that becouse Im creating a chatbot and that will be helpfull for me

**Comment by Hugo-ter-Doest** [comment]
I did some work on runtime language support. Please have a look at [this branch](https://github.com/Hugo-ter-Doest/natural/tree/setLanguage). There is a section at the top of the README about language support and I refactored the Porter stemmer.

**Comment by ran-j** [comment]
Yes, like that and how can I help to improve PT language functions ? (Sentiment,Stemmer,Tokenizer)

**Comment by LavrenkovNV** [comment]
how use russian language
Did you get the idea ?
